gensim/sklearn_integration/SklearnWrapperGensimLdaModel.py

```diff
@@ -43,33 +44,32 @@ def __init__(self, corpus=None, num_topics=100, id2word=None,
         # if no fit function is used , then corpus is given in init
         if self.corpus:
             models.LdaModel.__init__(
-                                     self, corpus=self.corpus, num_topics=self.num_topics, id2word=self.id2word,
-                                     chunksize=self.chunksize, passes=self.passes, update_every=self.update_every,
-                                     alpha=self.alpha, eta=self.eta, decay=self.decay, offset=self.offset,
-                                     eval_every=self.eval_every, iterations=self.iterations,
-                                     gamma_threshold=self.gamma_threshold, minimum_probability=self.minimum_probability,
-                                     random_state=self.random_state)
+                                self, corpus=self.corpus, num_topics=self.num_topics, id2word=self.id2word,
```

Comment:
Too much indentation. Hanging indent would have the line starting one indent more than line 46